src/Stratis.Bitcoin.Features.Notifications/BlockNotificationFeature.cs

```diff
@@ -51,7 +51,7 @@ public override void Start()
             this.chainState.ConsensusTip = this.chain.Tip;
         }
 
-        public override void Stop()
+        public override void Dispose()
```

Comment:
if you do this, you can just make sure to inject `/// <inheritdoc/>` everywhere ...